redis-snapshot-replicator/README.md

```diff
@@ -0,0 +1,45 @@
+#README
+
+## First steps before using the module
+
+In order to use this module, the Elasticache accountid must me added to the root s3 bucket. (the source bucket from where the replication starts)
+This way, the copy_snapshot command of Elasticache can do it's job.
+
+### Needed steps
+
+- Open the S3 console and go to newly created bucket (the source bucket from where the replication starts)
```

Comment:
Because I did not find it in TF :( I hope it is in there but I'm pretty sure it aint

Reply:
Yes you can add permissions to other AWS account by providing a bucket policy (`policy` argument in the `aws_bucket` resource). Here are some examples of such policy: https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html#example-bucket-policies-use-case-8

Reply:
As discussed in Slack, we would leave it like it is right now because the ACL part is an very old implementation and the long account id is not recognized in TF